.github/workflows/create-dev-tag.yml

```diff
@@ -52,15 +52,25 @@ jobs:
           git_push_gpgsign: false
 
       - name: Generate tag
-        run: echo "TAG=$(make ci.docker.run RUN_WHAT="make repo.dev.tag" | tail -1)" >> $GITHUB_ENV
+        run: echo "TAG=$(make repo.dev.tag | tail -1)" >> $GITHUB_ENV
         env:
           GITHUB_TOKEN: ${{ secrets.MESOSPHERECI_USER_TOKEN }}
 
       - name: Create tag
-        run: git tag -s -f -m "${{ env.TAG }}" ${{ env.TAG }}
+        run: |
+          ## Committer email does not match GPG private key email
+          ## So we need to set it expliclty here
+          git config user.email "ci-mergebot@d2iq.com"
+          git config user.name "ci-mergebot"
+          git tag -s -f -m "${{ env.TAG }}" ${{ env.TAG }}
 
       - name: Push tag
-        run: git push --force --tags origin ${{ env.TAG }}
+        run: |
+          ## Committer email does not match GPG private key email
+          ## So we need to set it expliclty here
+          git config --global user.email "ci-mergebot@d2iq.com"
+          git config --global user.name "ci-mergebot"
```

Comment:
Can't we use `git_config_global: true` in import gpg step?

Reply:
Or alternatively use `git_committer_name` and `git_committer_email` in the import gpg step?

Reply:
I did try this https://github.com/mesosphere/kommander-applications/actions/runs/6096111493/job/16540998959 but it was not signed for the same user/mail.